app/views/users/profile.html.erb

```diff
@@ -239,6 +245,18 @@
     alert("Your access token is: <%= @user.user.token %>");
   });
 
+$('#link_social_media').click(function(){
+   var sign = prompt("Enter Your Social Media User name like twitter:jywarren or instagram:unterbahn");
+    $.ajax({
+       method: "POST",
+       url: "http://localhost:3000/users/link/<%=@user.uid %> ",
+       data: {"sign": sign,"id":<%=@user.uid %>}
+})
+  .done(function( msg ) {
+    alert( "Data Saved succesfully" );
```

Comment:
Here, could we refresh the page, to display the new info? I think instead of using AJAX, we could in theory simply redirect through the user tag controller with `window.location = "..."` and the URL -- which is determined in the routes -- https://github.com/publiclab/plots2/blob/95e517f19881d9f3d2f7101cb60494b60bee9610/config/routes.rb#L194

This one accepts GET requests, so that may work!